### ISSUE TITLE
[Data|AIR] How can I distributed train a task with the dataset from 'to_torch()'?

### ISSUE THREAD

**Issue by guodashun** [ray-project/ray#27167]
### Description

According to the documentation, I can easily achieve the following two things:
**A. distributed load big data as ray dataset
B. distributed train a model by AIR**
However, a pre-request for **B** is that we have a _torch.Dataloader_ an we can convert it to distributed format by `train.torch.prepare_model(model)`, and we can only get the _IterableDataset_ format from `to_torch()`. If we pass the _IterableDataset_ directly into the _TorchTrainer_, we can only get the same result from the different workers because the dataloader returns the same data each batch, which makes the distributed training process meaningless.
In a nutshell, is there a bridge accross **ray dataset** to **AIR** for an end-to-end distributed train?

### Use case

End-to-end distributed train

**Comment by matthewdeng** [comment]
Hey @guodashun , thanks for the question!

In short, when integrating with Ray Datasets, the Dataset returned by [train.get_dataset_shard()](https://docs.ray.io/en/master/train/api.html#train-get-dataset-shard) will already be sharded, so each worker will only have the shard of data that it should be training on.

Here's some basic documentation about this: https://docs.ray.io/en/master/ray-air/check-ingest.html#configuring-ingest
Here's an example: https://docs.ray.io/en/master/ray-air/examples/torch_image_example.html

**Comment by guodashun** [comment]
@matthewdeng Thanks for your reply! Problem solved.
